launch/localization.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
```

Comment:
I assume we are putting the launch files for this node here because it's in a package that's not ours?